src/components/WalletModal/index.tsx

```diff
@@ -34,13 +32,47 @@ const WALLET_VIEWS = {
     PENDING: 'pending',
 }
 
+export async function checkAndSwitchMetamaskNetwork() {
+    // @ts-ignore
+    if (window.ethereum && window.ethereum.isMetaMask && typeof window.ethereum.request === 'function') {
+        // @ts-ignore
+        const chainId = await window.ethereum.request({ method: 'net_version' })
+        // Check if chain ID is same as REACT_APP_NETWORK_ID and prompt user to switch networks if not
+        if (chainId !== process.env.REACT_APP_NETWORK_ID) {
```

Comment:
@pi0neerpat I added the following:
- Added check for arbitrum one (arbitrum mainnet)
- There used to be toast functionality to warn users they were on the wrong network but that was removed somehow, so I added it back and changed it to be our color scheme so that on page refresh the user will still see they're on the wrong network. When on the right network the toast message is removed. I originally considered using the other "wrong network" modal rather than the toast, but I couldn't get this to work because this modal is used for multiple things (like displaying the wallet options). If the user tries to change their wallet they'll get that modal as well (see screenshots)
- Changed ETHERSCAN_PREFIXES constants to return names that work correctly with the error messaging, and changed the logic in helper.ts to make sure changing the prefixes doesn't break the etherscan links that are returned

## Screenshots

Connecting when on wrong network

https://github.com/open-dollar/od-app/assets/47253537/e54162cd-5f47-4bf3-82b5-349912a99054

Toast disappears once correct network selected

https://github.com/open-dollar/od-app/assets/47253537/5b791c4f-41eb-464c-a513-31bf2fa0dfab

User is also shown wrong networks modal in wallet menu 

https://github.com/open-dollar/od-app/assets/47253537/ee0f9d8b-92ea-44e9-a297-53e20469a73a